Lesson2/Lesson2_2/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson2_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            bool isInt(string value)
+            {
+                bool isCorrect = int.TryParse(value, out int res);
+                if (!isCorrect)
+                {
+                    Console.WriteLine("\r\nIncorrect value. Integer is needed.");
+                    Console.ReadLine();
+                }
+                return isCorrect;
+            }
+
+            Console.Clear();
+
+            Console.WriteLine("\r\nEnter first operand:");
+            var entered = Console.ReadLine();
+            if (!isInt(entered)) { return; }
+            int first = int.Parse(entered);
+
+            Console.WriteLine("\r\nEnter second operand:");
+            entered = Console.ReadLine();
+            if (!isInt(entered)) { return; }
+            int second = int.Parse(entered);
+
+            Console.WriteLine("\r\nEnter operation symbol:");
```

Comment:
@MariaOsminkina It's unclear to use, what symbols he could use, please change the input string